connectivity/http/BUILD

```diff
@@ -2,21 +2,14 @@ cc_library(
     name = "http",
     srcs = ["http.c"],
     hdrs = ["http.h"],
-    copts = select({
-        "//connectivity/mqtt:mqtt_enable": [
-            "-DMQTT_ENABLE",
-        ],
-        "//conditions:default": [],
-    }) + select({
-        "//storage:db_enable": ["-DDB_ENABLE"],
+    defines = select({
+        "//storage:db_enable": ["DB_ENABLE"],
```

Comment:
Do we need this since its' deps `"//accelerator/core:apis"` has defined DB_ENABLE?